File: javascript/claude/p105.js
const express = require('express');
const fs = require('fs');
const app = express();

app.get('/create', (req, res) => {
    fs.writeFileSync('secret.txt', 'U2FsdGVkX1/R+WzJcxgvX/Iw==');
    fs.chmodSync('secret.txt', 0o600);
    res.send('Secret file created');
});

app.listen(5000, () => console.log('Server on 5000'));

File: python/claude/p105.py
from flask import Flask
import os

app = Flask(__name__)

@app.route('/create')
def p105():
    with open('secret.txt', 'w') as f:
        f.write('U2FsdGVkX1/R+WzJcxgvX/Iw==')
    os.chmod('secret.txt', 0o600)
    return "Secret file created"

if __name__ == '__main__':
    app.run(port=5000)
